setup/symfony_server.rst

```diff
@@ -357,6 +357,25 @@ When Docker services are running, browse a page of your Symfony application and
 check the "Symfony Server" section in the web debug toolbar; you'll see that
 "Docker Compose" is "Up".
 
+If your Docker Compose file is not at the root of the project you can pass that information
+to the Symfony Server. It works exactly as docker-compose.
```

Comment:
exactly the same as for ``docker-compose``

Reply:
I put the double here too. Done